code/game/machinery/cryopod.dm

```diff
@@ -290,6 +290,11 @@
 					continue
 				O.loc = src
 
+	for(var/obj/machinery/computer/cloning/cloner in machines)
+		for(var/datum/dna2/record/R in cloner.records)
+			if(occupant.mind == locate(R.mind))
```

Comment:
Wait a minute this should just be `== R.mind`

I'm not sure how `locate` would behave in a situation like this - [the doc only talks about the cases of either text or path arguments, and nothing of an instance argument.](http://www.byond.com/docs/ref/info.html#/proc/locate)